iceberg_tools/graph/__init__.py

```diff
@@ -2,23 +2,38 @@
 import json
 from string import Formatter
 from typing import List, Iterator, Callable
+from glom import glom
 
+# import pyjq
 import fastjsonschema
 import jsonschema
-import pyjq
 import requests
 import re
 from jsonpointer import resolve_pointer
 
 from iceberg_tools.schema import extract_schemas, BASE_URI
 
-NESTED_OBJECTS = pyjq.compile(
-    '( paths | select(.[-1] == "$ref") ) as $p |   if getpath($p) != "Reference.yaml" then ( $p |  [.[] | tostring]  | join(".") ) else empty end')
-
 NESTED_OBJECTS_IGNORE = ['Identifier', 'Extension']
 
-REFERENCES = pyjq.compile(
-    '( paths | select(.[-1] == "$ref")) as $p |   if getpath($p) == "Reference.yaml" then ( $p | join(".") ) else empty end')
+
+def process_nested_schema_objects(data):
```

Comment:
Please add """Comment what this method does."""  docstring.

Reply:
Those functions probably shouldn't have gotten committed. Was an attempt at a Glom approach to iterate through a nested schema and return all glom paths to $ref fields. 

Didn't end up working and ended up going for a much more conventional approach for traversing a schema.